web/src/components/studies/AdminFeedbackSection.tsx

```diff
@@ -2,81 +2,66 @@ import Box from "../ui/Box";
 import { Textarea } from "../shared/exports";
 import styles from "./AdminFeedbackSection.module.css";
 import Button from "../ui/Button";
-import { useState } from "react";
+import { useEffect, useState } from "react";
 
 type FeedbackProps = {
   status: string;
-  feedback: string;
-  setFeedback: (feedback: string) => void;
-  handleUpdateStudyStatus: (status: string) => void;
+  feedbackFromStudy: string;
+  handleUpdateStudyStatus: (status: string, feedbackContent?: string) => void;
 };
 export default function AdminFeedbackSection(props: FeedbackProps) {
-  const { status, feedback, setFeedback, handleUpdateStudyStatus } = props;
-  const [showFeedbackForm, setShowFeedbackForm] = useState(false);
-  const [isCollapsing, setIsCollapsing] = useState(false);
-
-  const toggleShowFeedbackForm = () => {
-    // for smooth collapse
-    if (showFeedbackForm) {
-      setIsCollapsing(true);
-      setTimeout(() => {
-        setShowFeedbackForm(false);
-        setIsCollapsing(false);
-      }, 1400);
-    } else {
-      setShowFeedbackForm(true);
-    }
-  };
-
-  const handleFeedbackSubmit = () => {
-    handleUpdateStudyStatus("Rejected");
-    toggleShowFeedbackForm();
-  };
+  const { status, feedbackFromStudy, handleUpdateStudyStatus } = props;
+  const [feedback, setFeedback] = useState("");
 
   const handleFeedbackChange = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
     setFeedback(event.target.value);
   };
+
+  useEffect(() => {
+    const setFeedbackFromStudy = (feedbackFromStudy: string) => {
+      setFeedback(feedbackFromStudy);
+    };
+    if (feedbackFromStudy) {
+      setFeedbackFromStudy(feedbackFromStudy);
+    }
+  });
+
   return (
     <>
       {(status === "Pending" || status === "Rejected") && (
         <div>
-          <Button className={styles["approve-button"]} onClick={() => handleUpdateStudyStatus("Approved")}>
+          <Button className={styles["approve-button"]} onClick={() => handleUpdateStudyStatus("Approved", feedback)}>
             Approve Study
```

Comment:
when I click approve nothing happens.. expected? I'd expect the button to disappear(?)